amr-wind/physics/ConvectingTaylorVortex.cpp

```diff
@@ -103,8 +103,8 @@ ConvectingTaylorVortex::ConvectingTaylorVortex(const CFDSim& sim)
     pp.query("error_log_file", m_output_fname);
     {
         amrex::Real nu;
-        amrex::ParmParse pp("transport");
-        pp.query("viscosity", nu);
+        amrex::ParmParse pp2("transport");
+        pp2.query("viscosity", nu);
```

Comment:
In future we should just introduce a scope in the previous block. 

```c++
{
    amrex::ParmParse pp("CTV");
    pp.query("density", m_rho);
    pp.query("u0", m_u0);
    pp.query("v0", m_v0);
    pp.query("activate_pressure", m_activate_pressure);
    pp.query("error_log_file", m_output_fname);
}
```

Reply:
Yeah this makes more sense looking at it closer.